.gitattributes

```
@@ -0,0 +1,9 @@
+/.github export-ignore
+/tests export-ignore
+/tools export-ignore
+.editorconfig export-ignore
+.gitattributes export-ignore
+.gitignore export-ignore
+codecov.yml export-ignore
+phpstan.neon.dist export-ignore
+phpunit.xml.dist export-ignore
```

I think I’d prefer absolute paths for all ignored files.

```suggestion
/.editorconfig export-ignore
/.gitattributes export-ignore
/.github export-ignore
/.gitignore export-ignore
/codecov.yml export-ignore
/phpstan.neon.dist export-ignore
/phpunit.xml.dist export-ignore
/tests export-ignore
/tools export-ignore
```